apps/entry/popup.ts

```diff
@@ -22,19 +23,15 @@ function getCurrentTab(): Promise<chrome.tabs.Tab> {
 
 const exceptIkagadesitakaDiv = document.getElementById(
   "exceptIkagadesitakaDiv",
-) as HTMLDivElement;
+);
 const exceptIkagadesitakaButton = document.getElementById(
   "exceptIkagadesitakaButton",
-) as HTMLButtonElement;
-const searchInEnglishDiv = document.getElementById(
-  "searchInEnglishDiv",
-) as HTMLDivElement;
-const searchInEnglishButton = document.getElementById(
-  "searchInEnglishButton",
-) as HTMLButtonElement;
-const optionLink = document.getElementById("optionLink") as HTMLAnchorElement;
-
-searchInEnglishButton.addEventListener("click", async () => {
+);
+const searchInEnglishDiv = document.getElementById("searchInEnglishDiv");
+const searchInEnglishButton = document.getElementById("searchInEnglishButton");
+const optionLink = document.getElementById("optionLink");
```

Comment:
Please check for the existence of the element at this location.

Reply:
Done